languageFactories/language-factory-core/src/main/scala/cromwell/languages/util/ImportResolver.scala

```diff
@@ -12,86 +13,151 @@ import com.softwaremill.sttp.asynchttpclient.cats.AsyncHttpClientCatsBackend
 import common.Checked
 import common.transforms.CheckedAtoB
 import common.validation.ErrorOr._
+import common.validation.Checked._
 import common.validation.Validation._
 import cromwell.core.path.Path
+import java.nio.file.{Path => NioPath}
+
 import wom.core.WorkflowSource
 
 import scala.concurrent.duration._
 import scala.concurrent.Await
-import scala.util.Try
+import scala.util.{Failure, Success, Try}
 
 object ImportResolver {
```

Comment:
(optional, nice to have) I feel like this file would benefit from a breakup into trait + http + directory files

The distinctions between sections are not super clear